File: IEEE/Collect_Gold.cpp
#include<bits/stdc++.h>
using namespace std;

int get_gold(int id){
	return id;
}


int main(){

	int n,m;
	cin>>n>>m;
	
	vector<vector<int>> path(n);
	unordered_map<ULL,int> gold;
	bool visited[n];
	memset(visited,false,sizeof(visited));
	
	unsigend long long id;
	for(int i=0;i<n;i++){
		cin>>id;
		gold[id] = get_gold(id);

	}

	return 0;
	

}

File: IEEE/Collect_Gold.py
def get_gold(k):


    return k;
n,m = map(int,input().split())

gold = {}
dist = [0] * n
for i in range(n):
    dist[i] = [0]*n



for _ in range(n):
    k = int(input())
    gold[k] = get_gold(k)

for _ in range(m):
    x,y,d = map(int,input().split())

    dist[x].append(d)



print(n,m)
